### PR TITLE
22w07a stuff

### PR DESCRIPTION
**Pull request by apple502j**

### CODE REVIEW

File: mappings/net/minecraft/util/registry/SimpleRegistry.mapping [hunk 1]
```diff
@@ -46,3 +50,8 @@ CLASS net/minecraft/class_2370 net/minecraft/util/registry/SimpleRegistry
 		ARG 1 key
 	METHOD method_40277 (Lnet/minecraft/class_5321;)Lnet/minecraft/class_6880$class_6883;
 		ARG 1 key
+	METHOD method_40561 getEntriesList ()Ljava/util/List;
```
Comment (apple502j) [inline mappings/net/minecraft/util/registry/SimpleRegistry.mapping]: Q. Should we rename this to `getEntries` and rename existing `getEntries` to `getEntrySet`?

File: mappings/net/minecraft/world/chunk/UpgradeData.mapping [hunk 2]
```diff
@@ -23,8 +27,27 @@ CLASS net/minecraft/class_2843 net/minecraft/world/chunk/UpgradeData
...
 	METHOD method_12356 upgrade (Lnet/minecraft/class_2818;)V
 		ARG 1 chunk
+	METHOD method_40452 (Lnet/minecraft/class_1937;Lnet/minecraft/class_6759;)V
+		ARG 1 tick
+	METHOD method_40455 (Ljava/lang/String;)Ljava/util/Optional;
+		ARG 0 id
+	METHOD method_40456 addNeighborTicks (Lnet/minecraft/class_2487;Ljava/lang/String;Ljava/util/function/Function;Ljava/util/List;)V
+		ARG 0 nbt
+		ARG 1 key
+		ARG 2 nameToTypeFunction
```
Comment (enbrain) [inline mappings/net/minecraft/world/chunk/UpgradeData.mapping]: `Function` is redundant imo.
```suggestion
		ARG 2 nameToType
```

File: mappings/net/minecraft/world/tick/Tick.mapping [hunk 3]
```diff
@@ -23,11 +23,18 @@ CLASS net/minecraft/class_6759 net/minecraft/world/tick/Tick
...
 	METHOD method_39406 tick (Lnet/minecraft/class_2499;Ljava/util/function/Function;Lnet/minecraft/class_1923;Ljava/util/function/Consumer;)V
 		ARG 0 tickList
 		ARG 1 nameToTypeFunction
 		ARG 2 pos
 		ARG 3 tickConsumer
+	METHOD method_40558 (Lnet/minecraft/class_2487;Ljava/lang/Object;)Lnet/minecraft/class_6759;
+		ARG 1 type
+	METHOD method_40559 fromNbt (Lnet/minecraft/class_2487;Ljava/util/function/Function;)Ljava/util/Optional;
+		ARG 0 nbt
+		ARG 1 nameToTypeFunction
```
Comment (enbrain) [inline mappings/net/minecraft/world/tick/Tick.mapping]:
```suggestion
		ARG 1 nameToType
```

File: mappings/net/minecraft/util/registry/SimpleRegistry.mapping (hunk 1 above)
Comment (apple502j) [reply mappings/net/minecraft/util/registry/SimpleRegistry.mapping]:
```suggestion
	METHOD method_40561 getEntries ()Ljava/util/List;
```

**Review by Shnupbups — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**